core/services/pipeline/orm.go

```diff
@@ -410,6 +413,17 @@ WHERE pipeline_runs.id = batched_pipeline_runs.id`,
 		return errors.Wrap(err, "DeleteRunsOlderThan failed")
 	}
 
+	mark := time.Now()
```

Comment:
How about a more meaningful name here?
```suggestion
	deleteTS := time.Now()
```
:shrug: